docs/development/documentation.md

```diff
@@ -4,7 +4,18 @@
 
 `kops` uses cobra for it's CLI implementation. Each command should have the following help fields defined where possible:
 
-* `Short`: single sentence description of command.
-* `Long`: expanded description and usage of the command. The text from the `Short` field should be the first sentence in the `Long` field.
-* `Example`: example(s) of how to use the command. This field is formatted as a code snippet in the docs, so make sure if you have comments
-that these are written as a bash comment (e.g. `# this is a comment`).
+* `Short`: A short statement describing the command in the third person present, starting with a capital letter and ending without a period.
+  * Example: "Edits a cluster"
+
+* `Long`: An expanded description and usage of the command in the third person present tense, starting with a capital letter and ending with a period. The text from the `Short` field should be the first sentence in the `Long` field.
+Example:
+```
+Edits a cluster.
+
+This command changes the cloud specification in the registry.
+
+It does not update the cloud resources, to apply the changes use "kops update cluster".
+```
+
+* `Example`: Example(s) of how to use the command. This field is formatted as a code snippet in the docs, so make sure if you have comments that these are written as a bash comment (e.g. `# this is a comment`).
+
```

Comment:
What do we want in the root commands? For example, kops edit does not do a thing. "kops edit cluster" actually does something. Do we want to mention the sub commands? Let me see what kubectl does as wellz